content/docs/setup/kubernetes/sidecar-injection.md

```diff
@@ -203,8 +183,9 @@ sleep-776b7bcdcd-gmvnr   1/1       Running       0          2s
 [admissionregistration.k8s.io/v1beta1#MutatingWebhookConfiguration](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.10/#mutatingwebhookconfiguration-v1beta1-admissionregistration)
 configures when the webhook is invoked by Kubernetes. The default
 supplied with Istio selects pods in namespaces with label `istio-injection=enabled`.
-This can be changed by modifying the MutatingWebhookConfiguration in
-`install/kubernetes/istio-sidecar-injector-with-ca-bundle.yaml`.
+This can be changed by modifying the MutatingWebhookConfiguration with `kubectl edit mutatingwebhookconfiguration istio-sidecar-injector`. 
+
+> NOTE: The sidecar injector pod(s) should be restarted after modifying the mutatingwebhookconfiguration.
```

Comment:
@geeknoid do you have a NOTE preference - we should try to solve holistically as people seem keen to want to have that word note present - me included :)